portal/src/store/modules/filter-categories.js

```diff
@@ -133,7 +133,15 @@ export default {
   actions: {
     async getFilterCategories({ state, commit }) {
       const filters = getFiltersFromQuery(router.currentRoute.query)
-      if (
+      if (window.CATEGORY_FILTERS) {
```

Comment:
Van deze code heb je in de commit message aangegeven dat hij niet gebruikt werd. Ik wil je echter even wijzen op deze file: https://github.com/surfedushare/search-portal/blob/develop/service/surf/apps/materials/jinja2/portal/index.html

Dit is de index.html die gebruikt wordt op alle AWS instances. Je kunt deze index.html ook local gebruiken door http://localhost:8000 te bezoeken. Python heeft bij deze index.html een kans om data te injecten. We doen dat om een aantal redenen, maar ook om die filter categories te preloaden, zodat ze sneller beschikbaar zijn. Aanleiding voor deze optimalisatie was SEO verbeteren, want Google geeft strafpunten bij sites die grote "content shift" hebben.

Het stuk code wat jij hebt verwijderd laadde die filter categories in de store, zodat Vue er bij kon. Ik zou over het algemeen niet via http://localhost:8000 developen. De Python server moet herstarten en de static files opnieuw inladen voordat je wijzigingen ziet. Dat doet Vue's develop server dus een stuk beter dan de productie server die op 8000 draait in een container.